app/controllers/application_controller.rb

```diff
@@ -1,3 +1,14 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def after_sign_in_path_for(user)
+    case
+    when user.is_a?(Admin)
+      "/admins/#{current_admin.id}"
```

Comment:
I think you should use rails path helpers here to fill in the URL

something like `admins_path` and `partnering_organizations_path`